tasks/facial_landmark_detection/dataset.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import glob
+import os
+
+import numpy as np
+import torch
+from PIL import Image
+from torch.utils.data import Dataset
+
+import transforms
+
+class FaceLandmarkDataset(Dataset):
```

Comment:
would be nice if this code was flake8 clean:
```
pip install flake8
flake8 .
```

I have flake8 linting enabled in vscode so I see all the errors there, makes it easy to fix them.

Reply:
Do you know of any good auto formatter? Tried autoflake8 but it doesn't seem to satisfy flake8 which seems to be weird.

Reply:
You can use “black”. It’s supposed to follow our setup.cfg and follow flake8 guidelines.

Reply:
Let's do a full pass with that in the next PR.